zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/monitoring/BackupStats.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zookeeper.server.backup.monitoring;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class contains ZK backup related statistics
+ */
+public class BackupStats {
+  private static final Logger LOG = LoggerFactory.getLogger(BackupStats.class);
+
+  private int snapshotErrorCount = 0;
+  private long snapshotTimeSinceLastSuccessfulIteration = Long.MAX_VALUE;
+  private boolean snapshotBackupActive = false;
+  private long snapshotIterationDuration = 0;
+  private int snapshotBackupFilesCreatedPerIteration = 0;
+  private int txnLogErrorCount = 0;
+  private long txnLogTimeSinceLastSuccessfulIteration = Long.MAX_VALUE;
+  private boolean txnLogBackupActive = false;
+  private long txnLogIterationDuration = 0;
+  private int txnLogBackupFilesCreatedPerIteration = 0;
```

Comment:
Two things to consider:
1. Let's review the metric names to make sure they clearly mean what we want to measure.
2. Should we consider removing `txnLogBackupFilesCreatedPerIteration`, since the value of this is always going to be 1, if the iteration is successful?

Reply:
For 2, how about replacing it with numberOfTxnBackedUpLastIteration? But as in the design discussion, these metrics are nice-to-have, since we already have similar metrics in zookeeper server stats.

Reply:
On a second thought, I'll remove this metric for now, since it's not necessary for this phase.